### PR TITLE
Copy edit of all docs files.

### PR DESCRIPTION
**Pull request by eryland**
Copy edit of all documentation, as requested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -393,27 +393,26 @@ public class YourApplication extends Application {
...
 *Note:* 
-- If you are running in Admin mode on Android 6.0+, you will need to enable the “draw over other apps” permission via your OS settings. 
-- Dynamic configuration of Admin and User mode is supported.
+- If you are running in Admin mode on Android 6.0+, you must enable the “draw over other apps” permission through your OS settings. 
+- Dynamic configuration of both Admin and User mode is supported.
 
 **You have now successfully integrated the codeless Thunderhead SDK for Android.**
 
-## Additional features
+## Additional Features
```
Comment (alexandern17) [inline README.md:409]: Need to lowercase "features" as we decided to go with the format of only capitalizing the first letter in headers with the exception of Thunderhead terms like Interaction, Properties, Admin Mode, User Mode, Touchpoint, SalesForce, Interaction Studio, One, Tenant, Space, Activity, Activity Type, etc.
and all Android terms i.e. (AndroidManifest, Java)

File: README.md [hunk 2]
```diff
@@ -905,10 +898,10 @@ One.setAutomaticInteractionCallback(new OneInteractionPath(URI.create(TestConsta
 });
 ```
 
-The response can be passed to the `processResponse` method as shown above. By calling this method the response is returned to the SDK to process, attaching any capture, track, or optimize instructions to the Interaction.
+The response can be passed to the `processResponse` method, as shown above. By calling this method the response is returned to the SDK to process, attaching any activity capture, trattribute capture, or optimize instructions to the Interaction.
```
Comment (alexandern17) [inline README.md:901]: trattribute typo

File: README.md [hunk 3]
```diff
@@ -1054,7 +1049,7 @@ private void showVariants() {
 
 ### Ability to whitelist identity transfer links
 
-The SDK will append a `one-tid` url parameter to all links opened from a mobile app. If you would like to limit this behaviour, for the SDK to only append a `one-tid` to a specific set of links, you can whitelist the links to which the SDK should append a `one-tid` by calling the `java.net.URI.setIdentityTransferLinksWhiteList` Kotlin extension function or `whitelistIdentityTransferLinks` Java method as shown below:
+The SDK appends a `one-tid` URL parameter to all links opened from a mobile app. To limit this behaviour, and allow the SDK to append a `one-tid` URL parameter only to a specific set of links, whitelist that set of links by calling the `java.net.URI.setIdentityTransferLinksWhiteList` Kotlin extension function or `whitelistIdentityTransferLinks` Java method, as shown below:
```
Comment (alexandern17) [inline README.md:1052]: `URL`

File: README.md [hunk 4]
```diff
@@ -1109,12 +1104,12 @@ HashSet<URI> whitelist = One.getIdentityTransferLinksWhiteList()
...
-- Setting the whitelist to an `empty list` or `null` will clear the existing whitelist.
+- When a link is whitelisted, a `one-tid` is appended only to the whitelisted links.
+- Setting the whitelist to an `empty list` or `null` clears the existing whitelist.
 
 ### Ability to blacklist identity transfer links
 
-The SDK will append a `one-tid` url parameter to all links opened from a mobile app. If you would like to limit this behaviour, for the SDK to only append a `one-tid` to a specific set of links, you can blacklist the links to which the SDK should not append a `one-tid` by calling the `java.net.URI.setIdentityTransferLinksBlackList` Kotlin extension function or `blacklistIdentityTransferLinks` Java method as shown below:
+The SDK appends a `one-tid` URL parameter to all links opened from a mobile app. To limit this behaviour, and allow the SDK to append a `one-tid` URL parameter only to a specific set of links, blacklist the links to which the SDK should not append a `one-tid` by calling the `java.net.URI.setIdentityTransferLinksBlackList` Kotlin extension function or `blacklistIdentityTransferLinks` Java method, as shown below:
```
Comment (alexandern17) [inline README.md:1112]: `URL`

File: README.md [hunk 5]
```diff
@@ -1272,9 +1267,9 @@ Once you have the `uriWithOneTid`, pass this into the method which handles the o
 
 ### Disable automatic outbound link tracking
 
-If the Orchestration Plugin was enabled, the SDK will automatically send an Interaction request to `/one-click` as a url is opened in a `WebView`, `CustomTab` or external browser to facilitate last click attribution.
+If the Orchestration Plugin is enabled, the SDK automatically sends an Interaction request to `/one-click` when a URL is opened in a `WebView`, `CustomTab` or external browser to facilitate last click attribution.
```
Comment (alexandern17) [inline README.md:1270]: `URL`

File: README.md [hunk 6]
```diff
@@ -1397,11 +1391,11 @@ One.setMessagingConfiguration(oneMessagingConfiguration);
 ```
 
 *Note:* 
-- When the Thunderhead SDK is the only push message provider in your application and you enable codeless push notification support, the SDK will automatically get the push token and handle receiving of push notifications on behalf of your app.
+- When the Thunderhead SDK is the only push message provider in your application and you enable codeless push notification support, the SDK automatically gets the push token and handles the receiving of push notifications on behalf of your app.
 
 ##### Configure push notifications with multiple push message SDKs
 
-When the Thunderhead SDK is integrated into an app that has multiple push message providers for Firebase, extra configuration is required. The Thunderhead SDK message APIs must be called from the service that receives the FCM token and FCM Message.  
+When the Thunderhead SDK is integrated into an app that has multiple push message providers for Firebase, extra configuration is required. you must call the Thunderhead SDK message APIs from the service that receives the FCM token and FCM Message.
```
Comment (alexandern17) [inline README.md:1398]: You

File: README.md [hunk 7]
```diff
@@ -1582,29 +1575,29 @@ One.setLogLevel(OneLogLevel.FRAMEWORK);
...
 ```
 
 *Note:* 
-- This method removes `tid` from local storage only.
-- For instructions on how completely remove a user's data from Thunderhead ONE or Salesforce Interaction Studio - see our [api documentation](https://thunderheadone.github.io/one-api/#operation/delete).
+- Removes the stored `tid` only from local storage.
+- For instructions on how completely remove a user's data from Thunderhead ONE or Salesforce Interaction Studio, see our [API Documentation](https://thunderheadone.github.io/one-api/#operation/delete).
 
-## Further integration details 
+## Further Integration Details
```
Comment (alexandern17) [inline README.md:1596]: as mentioned above, only capitilizing first word in titles.  please revert this

File: README.md [hunk 8]
```diff
@@ -1618,13 +1611,13 @@ classpath 'com.thunderhead.android:orchestration-plugin:1.0.1'
 apply plugin: 'com.thunderhead.android.orchestration-plugin'
 ```
 
-## Troubleshooting guide
+## Troubleshooting Guide
 [Troubleshooting guide](TROUBLESHOOTING-GUIDE.md)
 
-## Questions or need help
+## Questions or Need Help
```
Comment (alexandern17) [inline README.md:1617]: revert

File: README.md [hunk 9]
```diff
@@ -1618,13 +1611,13 @@ classpath 'com.thunderhead.android:orchestration-plugin:1.0.1'
 apply plugin: 'com.thunderhead.android.orchestration-plugin'
 ```
 
-## Troubleshooting guide
+## Troubleshooting Guide
```
Comment (alexandern17) [inline README.md:1614]: revert

File: README.md [hunk 10]
```diff
@@ -1618,13 +1611,13 @@ classpath 'com.thunderhead.android:orchestration-plugin:1.0.1'
...
 [Troubleshooting guide](TROUBLESHOOTING-GUIDE.md)
 
-## Questions or need help
+## Questions or Need Help
 
 ### Salesforce Interaction Studio support
 _For Salesforce Marketing Cloud Interaction Studio questions, please submit a support ticket via https://help.salesforce.com/home_
 
-### Thunderhead ONE support
+### Thunderhead ONE Support
```
Comment (alexandern17) [inline README.md:1622]: revert

File: README.md [hunk 11]
```diff
@@ -1456,13 +1450,12 @@ Do not forget to register the customer service (if required) that calls the Thun
...
 
-The Thunderhead SDK will optimize your user's App experience by sending Push Notifications with _your_ application's icon when appropriate. In order to avoid the infinite crash
-loop that the above Android bug causes, the Thunderhead SDK will not show the message if a fallback *NON ADAPTIVE* icon is not set at initialization time on Api 26 devices. 
+The Thunderhead SDK will optimize your user's App experience by sending Push Notifications with _your_ application's icon when appropriate. In order to avoid the infinite crash loop that the above Android bug causes, the Thunderhead SDK will not show the message if a fallback *NON ADAPTIVE* icon is not set at initialization time on Api 26 devices. 
 Changing your application's icon to a non adaptive icon is not required and the fall back is **only required for Api 26**.
 
-The Thunderhead SDK will warn you at init if the icon has not been set by logging the `14019` error. See [Troubleshooting Guide](TROUBLESHOOTING-GUIDE.md)
+The Thunderhead SDK will warn you at init if the icon has not been set by logging the `14019` error. For more information, see the [Troubleshooting Guide](TROUBLESHOOTING-GUIDE.md)
```
Comment (alexandern17) [inline README.md:1458]: [Troubleshooting guide]

File: README.md (hunk 1 above)
Comment (eryland) [reply README.md:409]: Good to know, but does put you out of step with the guidelines followed by the company website and all other public documentation (including the Online Help).

File: README.md (hunk 3 above)
Comment (eryland) [reply README.md:1052]: Not sure what this comment means, Alex. URL is an acronym and, as such, should always be uppercase. Same with URI.

Comment (alexandern17) [reply README.md:1052]: Oh sorry, the comment formatted the change I was suggesting. 
Need some ticks ` around the URL text, so in Github it shows like `URL`

File: README.md (hunk 1 above)
Comment (alexandern17) [reply README.md:409]: Good point.  Maybe we need to have another discussion on this so we can be consistent on the formatting in all documentaiton @andreipop

File: README.md [hunk 12]
```diff
@@ -282,24 +282,24 @@ repositories {
 
 ```
 
-For further documentation on the `orchestration-plugin` please see the [reference docs](ORCHESTRATION-PLUGIN-README.md).
+For further documentation on the `orchestration-plugin` please see the [Orchestration Plugin Readme](ORCHESTRATION-PLUGIN-README.md).
 
-## Use the codeless Thunderhead SDK for Android
+## Use the Codeless Thunderhead SDK for Android
```
[the remark this answers is not in the document]
Comment (eryland) [reply README.md:287]: Done

File: README.md [hunk 13]
```diff
@@ -308,23 +308,23 @@ Included in the Thunderhead SDK's AndroidManifest.xml are the following permissi
 ### Configure and reconfigure the SDK
 
 You can configure and reconfigure the SDK as many times as necessary. 
-* The SDK does not support partial, or piecemeal, configuration. All parameters must be provided, either all valid or invalid (`empty string` or `null`).  
-* When configured with invalid parameters, it will set the SDK into an *unconfigured* state.
+* The SDK does not support partial, or piecemeal, configuration. You must provide all parameters, either all valid or invalid (`empty string` or `null`).  
+* When configured with invalid parameters, the SDK is initialized in an *unconfigured* state.
```
[the remark this answers is not in the document]
Comment (eryland) [reply README.md:312]: Done.

File: README.md (hunk 1 above)
Comment (eryland) [reply README.md:409]: Done from a docs point of view, but we do need to have that conversation to ensure consistancy across the the brand.

File: README.md (hunk 2 above)
Comment (eryland) [reply README.md:901]: Done.

File: README.md (hunk 3 above)
Comment (eryland) [reply README.md:1052]: Done

File: README.md (hunk 7 above)
Comment (eryland) [reply README.md:1596]: Done.

File: README.md (hunk 8 above)
Comment (eryland) [reply README.md:1617]: Done.

File: README.md (hunk 9 above)
Comment (eryland) [reply README.md:1614]: Done

File: README.md (hunk 10 above)
Comment (eryland) [reply README.md:1622]: Done.

File: README.md (hunk 11 above)
Comment (eryland) [reply README.md:1458]: Done

File: README.md (hunk 6 above)
Comment (alexandern17) [reply README.md:1398]: Bump - please capitalize You

File: README.md (hunk 5 above)
Comment (alexandern17) [reply README.md:1270]: Add ticks

File: README.md (hunk 3 above)
Comment (alexandern17) [inline README.md]: Add ticks around URL, two instances here.

File: README.md (hunk 4 above)
Comment (alexandern17) [reply README.md]: Missed adding ticks to the URL here, two of them.

File: README.md (hunk 5 above)
Comment (andreipop) [reply README.md:1270]: change URL to `URL`

File: README.md [hunk 14]
```diff
@@ -1330,20 +1325,19 @@ try {
...
 ### Enable push notifications
 
-To receive push notifications from Thunderhead ONE or Salesforce Interaction Studio, Firebase Cloud Messaging (FCM) must be configured by following the FCM setup instructions. 
-At minimum the app must be configured in Firebase and the `google-services.json` needs to be in the root of the app project.
+To receive push notifications from Thunderhead ONE or Salesforce Interaction Studio, configure Firebase Cloud Messaging (FCM) by following the FCM setup instructions. At a minimum the app must be configured in Firebase and the `google-services.json` needs to be in the root of the app project.
 
 #### Minimum Gradle configuration 
 
-To use the codeless push notifications functionality without using FCM directly, you need to at least have the `google-services` plugin applied to your app build.gradle: 
+To use the codeless push notifications functionality without using FCM directly, you must at least have the `google-services` plugin applied to your app build.gradle:
```
Comment (andreipop) [inline README.md:1340]: `build.gradle`

File: README.md [hunk 15]
```diff
@@ -1330,20 +1325,19 @@ try {
...
-To receive push notifications from Thunderhead ONE or Salesforce Interaction Studio, Firebase Cloud Messaging (FCM) must be configured by following the FCM setup instructions. 
-At minimum the app must be configured in Firebase and the `google-services.json` needs to be in the root of the app project.
+To receive push notifications from Thunderhead ONE or Salesforce Interaction Studio, configure Firebase Cloud Messaging (FCM) by following the FCM setup instructions. At a minimum the app must be configured in Firebase and the `google-services.json` needs to be in the root of the app project.
 
 #### Minimum Gradle configuration 
 
-To use the codeless push notifications functionality without using FCM directly, you need to at least have the `google-services` plugin applied to your app build.gradle: 
+To use the codeless push notifications functionality without using FCM directly, you must at least have the `google-services` plugin applied to your app build.gradle: 
 
 1. Add the Google Services Plugin to your classpath in the top-level build.gradle file, located in the root project directory, as shown below:
```
Comment (andreipop) [inline README.md:1342]: `build.gradle`

File: README.md [hunk 16]
```diff
@@ -69,7 +69,7 @@ on updating your existing SDK configuration.**
 Requires Gradle 5.2.1+
 
 1. Open your existing Android application in Android Studio.
-2. Include the Thunderhead SDK as a dependency into your project:
+2. Include the Thunderhead SDK as a dependency in your project:
 + Navigate to your **app-level** build.gradle file.
```
Comment (andreipop) [inline README.md:73]: `build.gradle`

File: MIGRATION-VERSION-6.md [hunk 17]
```diff
@@ -149,20 +141,16 @@ One.setMessagingConfiguration(oneMessagingConfiguration);
 ### Using static members
 
 Version 6.0.0 introduced the idea of static APIs to programmatically interact with the SDK.
-The [Thunderhead README](README.md#additional-features) `Additional Features` section provides steps to access functions
-available in the SDK.  A usage example is shown in this section of the migration guide.
+The [Thunderhead README](README.md#additional-features) `Additional Features` section provides steps to access function available in the SDK.  A usage example is shown in this section of the migration guide.
```
[the remark this answers is not in the document]
Comment (eryland) [reply MIGRATION-VERSION-6.md:144]: Done.

File: README.md [hunk 18]
```diff
@@ -2,13 +2,13 @@
...
-**For _migrating_ from version(s) <= 3.0.0 to version(s) 4.0.0+ of the Thunderhead SDK, please see the [plugin migration guide](ORCHESTRATION-PLUGIN-MIGRATION.md) 
+**To  _migrate_ from version(s) <= 3.0.0 to version(s) 4.0.0+ of the Thunderhead SDK, please see the [plugin migration guide](ORCHESTRATION-PLUGIN-MIGRATION.md) 
 for details on updating the required Gradle plugins.**
 
-**For _migrating_ from version(s) < 5.0.0 of the Thunderhead SDK to version(s) 5.0.0+, please see the [Java 8 migration guide](JAVA8-MIGRATION-GUIDE.md) for details
+**To _migrate_ from version(s) < 5.0.0 of the Thunderhead SDK to version(s) 5.0.0+, please see the [Java 8 migration guide](JAVA8-MIGRATION-GUIDE.md) for details
 on updating your app to be Java 8 compatible in order to use the Thunderhead SDK.**
 
-**For _migrating_ from version(s) < 6.0.0 of the Thunderhead SDK to version(s) 6.0.0+, please see the [Version 6 migration guide](MIGRATION-VERSION-6.md) for details
+**To _migrate__ from version(s) < 6.0.0 of the Thunderhead SDK to version(s) 6.0.0+, please see the [Version 6 migration guide](MIGRATION-VERSION-6.md) for details
```
[the remark this answers is not in the document]
Comment (eryland) [reply README.md:11]: Done.

File: README.md [hunk 19]
```diff
@@ -2,13 +2,13 @@
...
 on updating your existing SDK configuration.**
 
 ## Table of Contents
```
[the remark this answers is not in the document]
Comment (eryland) [reply README.md:14]: Done.

File: README.md (hunk 16 above)
Comment (eryland) [reply README.md:73]: Done.

File: README.md [hunk 20]
```diff
@@ -282,24 +282,24 @@ repositories {
...
 Enable your app to automatically recognize **Interactions** by executing the following steps.
 
-* Developer note: Android Studio `Instant Run` is not supported at this time and must be disabled.
+*Developer note:* Android Studio `Instant Run` is not currently supported and must be disabled.
 
-### The Thunderhead Application Manifest file permissions
+### Thunderhead Application Manifest file permissions
 
-Included in the Thunderhead SDK's AndroidManifest.xml are the following permissions which will be merged with your applications AndroidManifest.xml:
+The following permissions are included in the Thunderhead SDK's AndroidManifest.xml and will be merged with your applications AndroidManifest.xml:
```
[the remark this answers is not in the document]
Comment (eryland) [reply README.md:295]: Done.

File: README.md (hunk 5 above)
Comment (eryland) [reply README.md:1270]: Done.

File: README.md (hunk 14 above)
Comment (eryland) [reply README.md:1340]: Done.

File: README.md (hunk 15 above)
Comment (eryland) [reply README.md:1342]: Done.

File: README.md (hunk 6 above)
Comment (eryland) [reply README.md:1398]: Done.

**Pull request merged**